assignments/prototypes.js

```diff
@@ -7,22 +7,39 @@
   
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
-  
+
 /*
   === GameObject ===
   * createdAt
   * name
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+class GameObject {
+  constructor(attr) {
+    this.createdAt = attr.createdAt;
+    this.name = attr.name;
+    this.dimensions = attr.dimensions;
+  }
+  destroy() {
+    return `${this.name} was removed from the game. `;
+  }
+}
 /*
```

Comment:
I like how organized your class is, Rafal. You followed all the conventions creating it.